components/CountySearch.js

```diff
@@ -37,21 +36,24 @@ function CountySearchInput({
   );
 }
 
-export default function CountySearch({ searchRef }) {
-  const router = useRouter();
+export default function CountySearch({ searchRef, onSearch, setTopCountyOption }) {
+  const onInputChange = () => {
```

Comment:
Little hacky, but using `ref.current.items` was the only way I could figure out to grab latest list of options since the typeahead doesn't seem to expose this.